src/core/BufferGeometry.js

```diff
@@ -642,9 +642,49 @@ Object.assign( BufferGeometry.prototype, EventDispatcher.prototype, {
 
 	}(),
 
+	/**
+	 * Allows flat shading of faces on non-indexed buffer geometry
+	 */
 	computeFaceNormals: function () {
```

Comment:
Oh, I didn't realize that. I will add a comment to the function and remove this.